config/cars/mods/urd/urd_t5_maures_2018.ini

```diff
@@ -0,0 +1,92 @@
+[INCLUDE]
+INCLUDE=common/no_popup_lights.ini
+
+[LIGHTING]
+INTERIOR_FAKE_UPPER_SHADOW_HEIGHT=0.03
+INTERIOR_FAKE_UPPER_SHADOW_FADE=0.05
+EMISSIVE_HEADLIGHTS_MULT=3
+EMISSIVE_PARKINGLIGHTS_MULT=2
+EMISSIVE_BRAKELIGHTS_MULT=4
```

Comment:
These values are unreasonably high, please reconsider the usage of MULT= above 2.

I'd recommend testing with several mainstream post-process filters to see if the results align with reasonable brightness one would expect from a car such as this.